### PR TITLE
add logdb deployment

### PR DESCRIPTION
**Pull request by ticoann**
need to be tested don't merge (Review T0 cases)
Secrete file need to be updated
CENTRAL_LOGDB_URL=https://cmsweb.cern.ch/couchdb/wmstats_logdb
Also deployment script need to be updated
https://github.com/dmwm/deployment/commit/e987cf77f841ec391b309d5e683291eab6850b5a

### CODE REVIEW

File: src/python/WMCore/ReqMgr/CherryPyThreads/LogDBTasks.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+    def __init__(self, rest, config):
+
+        CherryPyPeriodicTask.__init__(self, config)
+
+    def setConcurrentTasks(self, config):
+        """
+        sets the list of functions which
+        """
+        self.concurrentTasks = [{'func': self.centalLogDBUpdate, 'duration': config.logDBUpdateDuration},
```
Comment (vkuznet) [inline src/python/WMCore/ReqMgr/CherryPyThreads/LogDBTasks.py]: please correct API name, centalLogDBUpdate to centralLogDBUpdate

File: src/python/WMCore/ReqMgr/CherryPyThreads/LogDBTasks.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                                {'func': self.logDBCleanUp, 'duration': config.logDBCleanDuration}]
+
+    def centalLogDBUpdate(self, config):
```
Comment (vkuznet) [inline src/python/WMCore/ReqMgr/CherryPyThreads/LogDBTasks.py]: please correct API name, centalLogDBUpdate to centralLogDBUpdate

File: src/python/WMCore/ReqMgr/CherryPyThreads/LogDBTasks.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        return
+    
+    def logDBCleanUp(self, config):
+        """
+        gather active data statistics
+        """
+        
+        logdb = LogDB(config.logdb_url, config.log_reporter, 
+                      config.central_logdb_url, logger = cherrypy.log)
+        logdb.delete_old_logs(config.logDBCleanDuration)
```
Comment (vkuznet) [inline src/python/WMCore/ReqMgr/CherryPyThreads/LogDBTasks.py]: please rename, I call it cleanup instead of delete_old_logs

**Comment by ticoann** [comment]
Thanks Valentin, I updated the names.

**Pull request merged**